backstop/tests/breadcrumbs.html

```diff
@@ -55,155 +54,324 @@
     <h1>Breadcrumbs</h1>
     <hr />
 
-    <section id="demo-all">
+    <h2>Generic anchor</h2>
+    <section id="demo-anchors">
 
-      <section id="demo-anchors">
+      <nav
+        class="iui-breadcrumbs"
+        aria-label="Breadcrumbs"
+      >
+        <button class="iui-button iui-borderless iui-dropdown">
+          <svg-folder
+            class="iui-button-icon"
+            aria-hidden
```

Comment:
wait, is it valid to have it without value? I remember there was something, that you can use that now, but I am not sure.

Reply:
`aria-hidden` without a value doesn't do anything. It needs to be `aria-hidden="true"`.

Reply:
Based off [this article](https://css-tricks.com/write-html-the-html-way-not-the-xhtml-way/#aa-examples=) I thought only `aria-hidden` would be needed, I can add `="true"`.

Reply:
wow that article is overflowing with bad advice